response.go

```diff
@@ -28,19 +28,43 @@ import (
 )
 
 type Response struct {
-	RspObj    interface{}
-	Exception error
-	//Attachments map[string]string
+	RspObj      interface{}
+	Exception   error
+	Attachments map[string]string
+}
+
+// Pls just use NewResponse to get a Response.
```

Comment:
change the comment to `NewResponse create a new Response`

Reply:
ok